sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -40,57 +40,57 @@ This scenario uses the [The OAuth 2.0 authorization code grant] flow to login in
 
-[//]: # "{x-version-update-start;com.azure.spring:azure-spring-boot-starter-active-directory;current}"
-```xml
-<dependency>
-    <groupId>com.azure.spring</groupId>
-    <artifactId>azure-spring-boot-starter-active-directory</artifactId>
-    <version>3.2.0</version>
-</dependency>
-<dependency>
-    <groupId>org.springframework.boot</groupId>
-    <artifactId>spring-boot-starter-oauth2-client</artifactId>
-</dependency>
-```
-[//]: # "{x-version-update-end}"
+    [//]: # "{x-version-update-start;com.azure.spring:azure-spring-boot-starter-active-directory;current}"
+    ```xml
+    <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>azure-spring-boot-starter-active-directory</artifactId>
+        <version>3.2.0</version>
+    </dependency>
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-oauth2-client</artifactId>
+    </dependency>
+    ```
+    [//]: # "{x-version-update-end}"
    
 * Step 3: Add properties in application.yml. These values should be got in [prerequisite].
-```yaml
-azure:
-  activedirectory:
-    tenant-id: xxxxxx-your-tenant-id-xxxxxx
-    client-id: xxxxxx-your-client-id-xxxxxx
-    client-secret: xxxxxx-your-client-secret-xxxxxx
-```
+    ```yaml
+    azure:
+      activedirectory:
+        tenant-id: xxxxxx-your-tenant-id-xxxxxx
+        client-id: xxxxxx-your-client-id-xxxxxx
+        client-secret: xxxxxx-your-client-secret-xxxxxx
+    ```
 
 * Step 4: Write your Java code:
 
-`AADWebSecurityConfigurerAdapter` contains necessary web security configuration for **aad-starter**.
-
- (A). `DefaultAADWebSecurityConfigurerAdapter` is configured automatically if you not provide one.
-
- (B). You can provide one by extending `AADWebSecurityConfigurerAdapter` and call `super.configure(http)` explicitly 
-in the `configure(HttpSecurity http)` function. Here is an example:
-<!-- embedme ../azure-spring-boot/src/samples/java/com/azure/spring/aad/AADOAuth2LoginConfigSample.java#L18-L29 -->
-```java
-@EnableWebSecurity
-@EnableGlobalMethodSecurity(prePostEnabled = true)
-public class AADOAuth2LoginConfigSample extends AADWebSecurityConfigurerAdapter {
-
-    @Override
-    protected void configure(HttpSecurity http) throws Exception {
-        super.configure(http);
-        http.authorizeRequests()
-            .antMatchers("/login").permitAll()
-            .anyRequest().authenticated();
+    `AADWebSecurityConfigurerAdapter` contains necessary web security configuration for **aad-starter**.
+
+     (A). `DefaultAADWebSecurityConfigurerAdapter` is configured automatically if you not provide one.
+    
+     (B). You can provide one by extending `AADWebSecurityConfigurerAdapter` and call `super.configure(http)` explicitly 
+    in the `configure(HttpSecurity http)` function. Here is an example:
+    <!-- embedme ../azure-spring-boot/src/samples/java/com/azure/spring/aad/AADOAuth2LoginConfigSample.java#L18-L29 -->
```

Comment:
Could you please update it to related code in sample project, and delete in `src/sample/java`.